### PR TITLE
WEL close temporary sites

### PR DESCRIPTION
**Pull request by mnaguit**
Close recorders (CUSP3D and ETNA 2)
Close sites with location code 1X, 21 and 22 (1Y is not on the list)
Close streams at LC 1X, 21 and 22 (1Y is not on the list)

### CODE REVIEW

**Comment by mnaguit** [comment]
To add to the clarifications and to speed up the processing of this PR, who will do the necessary adjustments in the metadata as enumerated by Jerome? @salichon @samgeonet @ozym

**Comment by salichon** [comment]
- [ ] 1X channel is sorted (PR is okay)
- [ ] 1Y needs to be sorted 
- [ ] 22 strong motion is sorted
- [ ] 21 strong motion is sorted

**Comment by samgeonet** [comment]
@salichon 1Y will be a lennartz either a Mk 2 or a Mk 3 depending on what 1X is

**Comment by mnaguit** [comment]
Thanks @ozym . I'll Natalie in this.
@nbalfour

**Comment by mnaguit** [comment]
Per discussion with @nbalfour and @salichon, I will modify the related csv files following Mark's suggestions. 

FYI @ozym and @samgeonet

**Comment by ozym** [comment]
found data for WEL 21 in 1992

**Comment by salichon** [comment]
@mnaguit perhaps you can close WEL 21 instead ?

**Comment by mnaguit** [comment]
Please review again @ozym @salichon @samgeonet 
Thanks!

**Comment by mnaguit** [comment]
I need a lot of chocolate Easter eggs to recover from this @salichon :)

**Comment by salichon** [comment]
Question to be consistent:
does this guys to be cleaned too? 
> WEL,11,Wellington Dominion Observatory,Test 1,2014-07-25T00:05:00Z,2014-11-03T02:00:00Z
> WEL,12,Wellington Dominion Observatory,Test 2,2014-11-03T02:10:00Z,2015-03-24T00:00:00Z

there is no such a data in the archive

**Comment by mnaguit** [comment]
Hi @ozym . Are you happy with the changes in this PR? Thanks.

File: install/connections.csv [hunk 1]
```diff
@@ -897,10 +897,9 @@ WEL,11,Wellington Dominion Observatory,SNARE Network,1986-09-22T00:00:00Z,1990-1
 WEL,11,Wellington Dominion Observatory,Short Period,1994-11-25T23:58:00Z,1995-06-22T18:31:00Z
 WEL,11,Wellington Dominion Observatory,Test 1,2014-07-25T00:05:00Z,2014-11-03T02:00:00Z
 WEL,12,Wellington Dominion Observatory,Test 2,2014-11-03T02:10:00Z,2015-03-24T00:00:00Z
-WEL,1X,Wellington Dominion Observatory,Test,2016-04-26T03:00:01Z,9999-01-01T00:00:00Z
 WEL,20,Wellington Dominion Observatory,,2003-04-08T00:00:01Z,9999-01-01T00:00:00Z
 WEL,21,Wellington Dominion Observatory,Earss,1990-10-03T14:20:00Z,2003-09-19T23:10:00Z
-WEL,21,Wellington Dominion Observatory,Test,2016-04-26T03:00:01Z,9999-01-01T00:00:00Z
+WEL,21,Wellington Dominion Observatory,Test,2016-04-26T03:00:01Z,2019-04-10T00:30:00Z
```
Comment (ozym) [inline install/connections.csv]: This is the test site so this line needs to be removed as well.

File: install/dataloggers.csv [hunk 2]
```diff
@@ -668,7 +668,7 @@ Kinemetrics,Q330HR/6,3104,Matahi Valley,,2016-11-10T22:42:01Z,2017-09-14T01:35:0
 Kinemetrics,Q330HR/6,3105,Birch Farm,,2009-11-19T01:00:01Z,9999-01-01T00:00:00Z
 Kinemetrics,Q330HR/6,3106,Omahuta,,2010-02-24T00:00:01Z,9999-01-01T00:00:00Z
 Kinemetrics,Q330HR/6,3107,Birch Farm,,2008-11-09T22:13:01Z,2009-11-19T01:00:00Z
-Kinemetrics,Q330HR/6,3107,Wellington Dominion Observatory,Test,2017-07-20T22:20:01Z,9999-01-01T00:00:00Z
+Kinemetrics,Q330HR/6,3107,Wellington Dominion Observatory,Test,2017-07-20T22:20:01Z,2017-12-20T21:30:00Z
```
Comment (ozym) [inline install/dataloggers.csv]: This was a test instrument, also needs to be removed.

File: install/dataloggers.csv [hunk 3]
```diff
@@ -781,6 +781,7 @@ Kinemetrics,Q330S/6,6139,Awhitu Peninsula,,2018-07-05T03:30:00Z,9999-01-01T00:00
 Kinemetrics,Q330S/6,6140,Waitaha Valley,,2018-07-08T23:30:00Z,9999-01-01T00:00:00Z
 Kinemetrics,Q330S/6,6141,Pawanui,,2016-07-22T00:30:01Z,9999-01-01T00:00:00Z
 Kinemetrics,Q330S/6,6142,Moumoukai,,2018-07-05T01:00:00Z,9999-01-01T00:00:00Z
+Kinemetrics,Q330S/6,6143,Wellington Dominion Observatory,Test,2017-12-20T22:00:00Z,2019-04-10T00:30:00Z
```
Comment (ozym) [inline install/dataloggers.csv]: This too needs to be removed, was another test install.

File: install/recorders.csv [hunk 4]
```diff
@@ -380,7 +380,7 @@ CSI,CUSP3D,CUSP3D,42104,MAGS,20,269,0,0,2014-08-18T00:03:00Z,2014-08-18T00:04:00
 CSI,CUSP3D,CUSP3D,42104,MAGS,20,311,0,0,2014-08-18T00:05:00Z,9999-01-01T00:00:00Z
 CSI,CUSP3D,CUSP3D,42105,CUBS,20,23,0,0,2013-08-22T01:30:00Z,2015-10-16T00:00:00Z
 CSI,CUSP3D,CUSP3D,42105,WSTS,20,169,0,0,2016-12-02T02:30:00Z,2017-03-14T14:00:00Z
-CSI,CUSP3D,CUSP3D,42105,WEL,22,0,0,0,2017-05-31T03:26:00Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42105,WEL,22,0,0,0,2017-05-31T03:26:00Z,2019-04-10T00:30:00Z
```
Comment (ozym) [inline install/recorders.csv]: This needs to be removed, was a TEST install at site 22.

File: install/recorders.csv [hunk 5]
```diff
@@ -1098,7 +1098,7 @@ Kinemetrics,FBA-ES-T-ETNA-2,ETNA 2,100443,GLOS,20,0,0,0,2017-06-12T04:25:00Z,201
 Kinemetrics,FBA-ES-T-ETNA-2,ETNA 2,100443,GLOS,20,0,0,0,2017-06-12T04:28:00Z,9999-01-01T00:00:00Z
 Kinemetrics,FBA-ES-T-ETNA-2,ETNA 2,100559,TENS,20,0,0,0,2017-11-14T23:00:00Z,9999-01-01T00:00:00Z
 Kinemetrics,FBA-ES-T-ETNA-2,ETNA 2,100563,SDNS,20,0,0,0,2018-08-14T21:45:00Z,9999-01-01T00:00:00Z
-Kinemetrics,FBA-ES-T-ETNA-2,ETNA 2,100564,WEL,21,0,0,0,2017-05-31T03:26:00Z,9999-01-01T00:00:00Z
+Kinemetrics,FBA-ES-T-ETNA-2,ETNA 2,100564,WEL,21,0,0,0,2017-05-31T03:26:00Z,2019-04-10T00:30:00Z
```
Comment (ozym) [inline install/recorders.csv]: This needs to be removed, was a TEST install at site 21.

**Comment by ozym** [comment]
@mnaguit have added a review with more details.

**Comment by mnaguit** [comment]
Thank you @ozym . I have removed all test sites in connections and dataloggers and test site 22 and 21 (only the latest entry) as well.

Kindly do another round of checking @ozym @salichon and @samgeonet .
Many thanks!

**Pull request merged**